pkg/image/nbdkit.go

```diff
@@ -228,6 +236,30 @@ func getVddkPluginPath() NbdkitPlugin {
 	return NbdkitVddkPlugin
 }
 
+// Extra VDDK configuration options are stored in a ConfigMap mounted to the
+// importer pod. Just look for the first file in the mounted directory, and
```

Comment:
`Just look for the first file` just note we need to be sure to document this so user does not chain the configs to separate configmaps.